### PR TITLE
creates task spec in readme

### PR DESCRIPTION
**Pull request by rickfoxcroft**
Frontend tech task spec. Have a look at the readme. Is there sufficient detail?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
...
+
+You will be given the email address of a member of the team, and should email this person with a link to your solution at least 24 hours before the scheduled technical interview. You're also encouraged to email them with questions at any time while working on this task.
+
+**Please don't spend more than a few hours on this task.**
+
+It's not a trick question and we're not expecting anything too elaborate. It's a small self-contained exercise to help us get to know you through your code - how you break down a problem, how you structure your code, you're approach to styling and solving design problems, and how you navigate problem solving with your potential future team-mates.
+
+We'll be looking for well-structured, clear, maintainable code with reasonable performance characteristics. We will not be looking for micro-optimisations or for every single detail to be carefully polished.
+
+We'll be looking for well-structured, clear, maintainable code with reasonable performance characteristics. We would also like to see a responsive, extensible and accessible style implementation.
```
Comment (jayvdb) [inline README.md]: first sentence is duplicated from above paragraph

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+
+We will not be looking for micro-optimisations or for every single detail to be carefully polished.
```
Comment (jayvdb) [inline README.md]: dup of above

File: books.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        "imageUrl": ""
+    },
+    {
+        "author":"Dr Jason Fung",
+        "title": "The Cancer Code",
+        "publisherSummary": "In The Cancer Code, Dr Jason Fung offers a revolutionary new understanding of this invasive, often fatal disease – what it is, how it manifests and why it is so challenging to treat. In this rousing narrative, Dr Fung identifies the medical community’s many missteps in cancer research – in particular, its focus on genetics, or what he terms the ‘seed’ of cancer, at the expense of examining the ‘soil,’ or the conditions under which cancer flourishes. Dr Fung – whose groundbreaking work in the treatment of obesity and diabetes has won him international acclaim – suggests that the primary disease pathway of cancer is caused by the dysregulation of insulin. In fact, obesity and type 2 diabetes significantly increase an individual’s risk of cancer.",
```
Comment (jayvdb) [inline books.json]: contains non-ascii. not a problem; jut checking we want to be doing this..?

**Review by jayvdb — DISMISSED** [review]

File: README.md [hunk 4]
```diff
@@ -1,2 +1,27 @@
...
+
+It's not a trick question and we're not expecting anything too elaborate. It's a small self-contained exercise to help us get to know you through your code - how you break down a problem, how you structure your code, you're approach to styling and solving design problems, and how you navigate problem solving with your potential future team-mates.
+
+We'll be looking for well-structured, clear, maintainable code with reasonable performance characteristics. We would also like to see a responsive, extensible and accessible style implementation. We will not be looking for micro-optimisations or for every single detail to be carefully polished.
+
+## Task Requirements
+
+All of the assets and style details required should be available in the [Figma prototype inspection](https://www.figma.com/file/krV1Uw4Dh84NUzIJg0pDUH/Coding-Test---Draggable-Component?node-id=0%3A1).
+
+Use the `books.json` file located in the project root directory to fetch data required for the app. You will have to export the images for each book from the Figma prototype inspection and host them alongside your solution. Add the image location to the json file and refer to the images accordingly in your code.
```
Comment (StuartW-AI) [inline README.md:27]: Either add a note that the JSON does not contain any cover art images as shown in Figma or ideally update the JSON file to link to an image, e.g. for Homo Deus use the Wikipedia image, https://en.wikipedia.org/wiki/Homo_Deus:_A_Brief_History_of_Tomorrow#/media/File:Homo_Deus.jpg

File: books.json [hunk 5]
```diff
@@ -0,0 +1,44 @@
+[
+    {
+        "author": "Yuval Noah Harari",
+        "title": "Sapiens",
+        "publisherSummary": "The Sunday Times best seller. Earth is 4.5 billion years old. In just a fraction of that time, one species among countless others has conquered it. Us. We are the most advanced and most destructive animals ever to have lived. What makes us brilliant? What makes us deadly? What makes us sapiens? In this bold and provocative audiobook, Yuval Noah Harari explores who we are, how we got here, and where we're going.",
+        "imageUrl": ""
```
Comment (StuartW-AI) [inline books.json:6]: https://en.wikipedia.org/wiki/Sapiens:_A_Brief_History_of_Humankind#/media/File:Sapiens,_A_Brief_History_of_Humankind_-_Random_House_Vintage_Paperback_Edition.jpg

**Review by StuartW-AI — APPROVED** [review]

**Review by jayvdb — APPROVED** [review]

**Pull request merged**